resolwe_bio/processes/differential_expression/dexseq.yml

```diff
@@ -0,0 +1,132 @@
+# ================================
+# Differential Expression - DEXSeq
+# ================================
+---
+
+- slug: differentialexpression-dexseq
+  name: DEXSeq
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/rnaseq:2.1.1
+  data_name: 'Differential expression (case vs. control)'
+  version: 0.0.1
+  type: data:differentialexpression:dexseq
```

Comment:
We have a [meta process](data:differentialexpression) of `data:differentialexpression` type that defines the required fields for the `data:differentialexpression:*` processes. Here, we are missing the `de_json` and `de_file` fields. @JureZmrzlikar how come this was not reported by the failing tests? Were the tests implemented just for migrations? 

Are the `de_json` and `de_file` fields even relevant for this analysis? Is the output of DEX-seq viewable in Volcano plot?

Reply:
For these outputs we need to parse the dexseq output table. We have several options how to do that:

1.) We change the https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/tools/parse_diffexp.py tool to run on python3:

1.1.) That would require changing some of the differential expression's docker images to `resolwebio/rnaseq:2.1.1` as it has numpy and pandas installed in python3. For some processes (edgeR, limma) we would have to create a new version of rnaseq docker image because they only live in the legacy image which has python2 numpy and pandas. I prefer this option as we will have to change processes in the future anyway. The only problem is that Ubuntu 18.04 is not "stable" yet.

1.2.) Or we run the script from the older processes with python2.

2.) We keep the parse_diffexp.py script as it is and run it from dexseq process with python3.